### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by RedstoneWizard08**
**Description of the change**

Adds support for armhf (32-bit arm or `linux/arm/v7` in docker).

**Benefits**

More supported platforms.

**Possible drawbacks**

More build times?

**Applicable issues**

This is working on #7305 in `bitnami/charts` but it isn't done.

**Additional information**

This is not done. This is currently a pull request so that I can see the changes made and squash the commits and such. I'll post a comment when it is done.

### CODE REVIEW

**Comment by RedstoneWizard08** [comment]
https://github.com/RedstoneWizard08/minideb/actions/runs/1821260875

**Comment by RedstoneWizard08** [comment]
I believe that the buster amd64 run (https://github.com/RedstoneWizard08/minideb/runs/5134326204?check_suite_focus=true) was a success but couldn't push. That's probably because I don't have the secrets set, and I don't want to set them.

**Comment by RedstoneWizard08** [comment]
@DCO you are annoying.

I squashed all the commits so hopefully this works. Everything has compiled successfully so it should be able to push with the correct tokens. I don't have the tokens so it couldn't push, meaning that some jobs failed.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
+      - patch-1
   pull_request:
 
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
   schedule:
     - cron:  '0 0 * * *'
 env:
-  BASENAME: bitnami/minideb
+  BASENAME: redstonewizard08/minideb
```
Comment (javsalgar) [inline .github/workflows/main.yml:19]: Is this change expected?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
 name: CI
-
+ 
 # Controls when the action will run.
 on:
   # Triggers the workflow on push or pull request events but only for the master branch
   push:
     branches:
       - master
+      - patch-1
```
Comment (javsalgar) [inline .github/workflows/main.yml:11]: Could you explain this change?

File: .gitpod.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# This configuration file was automatically generated by Gitpod.
```
Comment (javsalgar) [inline .gitpod.yml:1]: What's this file for?

File: .github/workflows/main.yml (hunk 1 above)
Comment (RedstoneWizard08) [reply .github/workflows/main.yml:19]: It is until production, @javsalgar. This is just until it's merged - I did that for testing.

File: .github/workflows/main.yml (hunk 2 above)
Comment (RedstoneWizard08) [reply .github/workflows/main.yml:11]: That's just for testing. Before I merge, it'll be reverted.

File: .gitpod.yml (hunk 3 above)
Comment (RedstoneWizard08) [reply .gitpod.yml:1]: I edit in GitPod.

Comment (javsalgar) [reply .gitpod.yml:1]: Could you remove these extra files?

Comment (RedstoneWizard08) [reply .gitpod.yml:1]: I will.

**Comment by carrodher** [comment]
I'm closing this PR due to lack of updates, please feel free to reopen it if needed updating the code and applying the suggestions

**Pull request closed without merging**